pkg/utils/utils.go

```diff
@@ -30,3 +30,12 @@ func FindMatches(pattern string, text []byte) map[string]interface{} {
 	}
 	return configMap
 }
+
+func Contains[T comparable](s []T, e T) bool {
```

Comment:
We are back again with the contains things XD
We started using the `map[string]bool` to solve this, as it was much more proficient in golang languange, but so be it hehe